plugins/app-backend/src/service/router.ts

```diff
@@ -132,8 +133,15 @@ export async function createRouter(
   const staticRouter = Router();
   staticRouter.use(
     express.static(resolvePath(appDistDir, 'static'), {
-      setHeaders: res => {
-        res.setHeader('Cache-Control', CACHE_CONTROL_MAX_CACHE);
+      setHeaders: (res, path) => {
+        if (path === injectedConfigPath) {
+          logger.info(
+            `Serving in the injected Javascript file with max caching disabled`,
+          );
```

Comment:
Should probably remove this log line (especially since it doesn't say what path it was etc).